bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -467,3 +467,20 @@ def to_api_repr(self):
         if self.name is not None:
             resource['name'] = self.name
         return resource
+
+
+class QueryParametersProperty(object):
+    """Custom property type, holding query parameter instances."""
+
+    def __get__(self, instance, owner):
+        """Descriptor protocol:  accessor"""
+        if instance is None:
+            return self
+        return list(instance._query_parameters)
+
+    def __set__(self, instance, value):
+        """Descriptor protocol:  mutator"""
+        if not all(isinstance(u, AbstractQueryParameter) for u in value):
```

Comment:
By inspection, `value` is a sequence, and its items are expected to be instances of classes derived from `AbstractQueryParameter`.  Using a longer name for the placeholder iteration value doesn't add information, and makes the expression wrap, which then puts us into lint hell.

I will update the docstrings to make the sense more obvious.